src/main/kotlin/com/workos/directorysync/models/UserUpdate.kt

```diff
@@ -0,0 +1,87 @@
+package com.workos.directorysync.models
```

Comment:
Instead of creating a new model here and extending it in `UserUpdated`, we could directly define these fields in the `UserUpdated` model. Basically -- move everything here into the `UserUpdated` model.

Reply:
Refactored this to be a class within the UserUpdated model. 👍🏽